src/main/java/org/folio/sidecar/integration/keycloak/configuration/JsonWebTokenExpiry.java

```diff
@@ -25,7 +25,7 @@ public JsonWebTokenExpiry(KeycloakProperties keycloakProperties) {
 
   @Override
   public long expireAfterCreate(String s, JsonWebToken jsonWebToken, long currentTime) {
-    return SECONDS.toNanos(jsonWebToken.getExpirationTime()) - MILLISECONDS.toNanos(currentTimeMillis() + expireOffset);
+    return SECONDS.toNanos(jsonWebToken.getExpirationTime()) - MILLISECONDS.toNanos(currentTimeMillis()) - expireOffset;
```

Comment:
Why are we calling currentTimeMillis() here at all if we have currentTime as a parameter?

Reply:
actually this `currentTime` is not what we all thinking about, it is `System.nanoTime()`